assign-doi-datacite/src/test/java/no/unit/nva/doi/datacite/clients/OnlineTests.java

```diff
@@ -0,0 +1,51 @@
+package no.unit.nva.doi.datacite.clients;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.hamcrest.Matchers.nullValue;
+import static org.hamcrest.core.IsNot.not;
+import java.net.URI;
+import java.util.Map;
+import no.unit.nva.doi.datacite.clients.exception.ClientException;
+import no.unit.nva.doi.datacite.clients.models.Doi;
+import no.unit.nva.doi.datacite.config.DataCiteConfigurationFactory;
+import no.unit.nva.doi.datacite.config.DataCiteConfigurationFactoryForSystemTests;
+import no.unit.nva.doi.datacite.config.PasswordAuthenticationFactory;
+import no.unit.nva.doi.datacite.mdsclient.DataCiteConnectionFactory;
+import no.unit.nva.doi.datacite.models.DataCiteMdsClientSecretConfig;
+import nva.commons.utils.Environment;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+
+public class OnlineTests {
+
+    public static final URI DATACITE_DRAFT_DOI_REST_API = URI.create("https://api.test.datacite.org/dois");
+    private static final URI EXAMPLE_CUSTOMER_ID = URI.create("https://example.net/customer/id/4512");
+
+    @Test
+    @Tag("online")
```

Comment:
This test is disabled by default. It is here to be able to debug problems with Drafting a DOi